src/style.css

```diff
@@ -0,0 +1,86 @@
+:root {
+  --grey: #e1e7ed;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.form-item {
+  width: 100%;
+  height: 70px;
+  font-weight: 300;
+  border: 1px solid var(--grey);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  transition: 3s;
```

Comment:
as recommendation: add transition-properties to define what exactly you'd like to change to avoid using `all` default value